vendor/github.com/livepeer/lpms/core/lpms.go

```diff
@@ -30,7 +30,6 @@ type LPMS struct {
 	vidPlayer   *vidplayer.VidPlayer
 	vidListener *vidlistener.VidListener
 	workDir     string
-	httpAddr    string
```

Comment:
I don't think we should change anything in LPMS itself. It's good as it is, we just don't need to pass our `httpAddr` to it.

Reply:
I've tested and everything works as expected. Do we not want to change LPMS in case we in the future add something that does need the `httpAddr` field? Or is that a field that might be used for something else? 

So the only changes we make will be to go-livepeer?

Reply:
Agreed with @darkdarkdragon . The changes here should be for go-livepeer only. Any changes to LPMS should be proposed and implemented in a separate PR there. Even then, this type of change is not really necessary IMO, as it removes a feature that is still useful to LPMS by itself.

Reply:
Ah I see. I misunderstood. I've reverted the LPMS changes!